packages/docs/docs/tailwind.md

```diff
@@ -60,110 +60,59 @@ values={[
 <TabItem value="npm">
 
 ```bash
-npm i -D postcss-loader postcss postcss-preset-env tailwindcss autoprefixer css-loader style-loader
+npm i -D @remotion/tailwind postcss-loader postcss postcss-preset-env tailwindcss autoprefixer css-loader style-loader
 ```
 
   </TabItem>
 
   <TabItem value="yarn">
 
 ```bash
-yarn add -D postcss-loader postcss postcss-preset-env tailwindcss autoprefixer css-loader style-loader
+yarn add -D @remotion/tailwind postcss-loader postcss postcss-preset-env tailwindcss autoprefixer css-loader style-loader
 ```
 
   </TabItem>
   <TabItem value="pnpm">
 
 ```bash
-pnpm i -D postcss-loader postcss postcss-preset-env tailwindcss autoprefixer css-loader style-loader
+pnpm i -D @remotion/tailwind postcss-loader postcss postcss-preset-env tailwindcss autoprefixer css-loader style-loader
```

Comment:
Just wondering if we'd still need to ask users to install all the other deps 

```shell
pnpm i -D @remotion/tailwind postcss-loader postcss postcss-preset-env tailwindcss autoprefixer css-loader style-loader
```

instead of just

```shell
pnpm i -D @remotion/tailwind
```

since I saw all the other deps are `devDependencies` of the `@remotion/tailwind` package!

Reply:
Yeah it seems necessary, unfortunately. I was hoping their presence in devDependencies would avoid the need to do so, but that doesn't seem to be the case.

These are the errors I get trying to run `packages/example` with just `@remotion/tailwind`:

```
$ pnpm start

> @remotion/example@3.3.87 start /Users/rjackson/Code/remotion/packages/example
> remotion preview --props src/my-props.json

Server ready - Local: http://localhost:3000, Network: http://192.168.86.27:3000
ERROR in ./src/Video.tsx 1:0-21
Module not found: Error: Can't resolve 'style-loader' in '/Users/rjackson/Code/remotion/packages/example'
ERROR in ../cli/dist/previewEntry.js 9:0-31
Module not found: Error: Can't resolve 'style-loader' in '/Users/rjackson/Code/remotion/packages/example'
Built in 1275ms
```

Reply:
@rjackson @Just-Moh-it What if they were dependencies instead of devDependencies?
That would be totally valid!

Reply:
Darn I got excited, but that doesn't seem to work either.

I feel like it should work but I've had a whole prune, clean, reinstall, and rebuild, but it's still giving the same errors :(

```sh
$ pnpm store prune
$ pnpm clean
$ pnpm i
$ pnpm build
$ cd packages/example
$ pnpm start

> @remotion/example@3.3.87 start /Users/rjackson/Code/remotion/packages/example
> remotion preview --props src/my-props.json

Server ready - Local: http://localhost:3000, Network: http://192.168.86.27:3000
ERROR in ./src/Video.tsx 1:0-21
Module not found: Error: Can't resolve 'css-loader' in '/Users/rjackson/Code/remotion/packages/example'
ERROR in ../cli/dist/previewEntry.js 9:0-31
Module not found: Error: Can't resolve 'css-loader' in '/Users/rjackson/Code/remotion/packages/example'
Built in 2204ms
```

Reply:
Ah damn 😐 Maybe `require.resolve("css-loader")`?
That should resolve the `css-loader` package relative to `@remotion/tailwind`!

If that does not work, then okay, let's ship it like that.